content/en/internal/configuration-database/howto-faq/_index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "HOW TOs and FAQs"
+weight: 30
+description: >-
+     HOW TOs and FAQs.
+---
+
+## I get an "error 12227" message when accessing GOC portal with Mozilla/Firefox
+
+This happens when no certificate has been uploaded to your browser. Refer to the
+"Access to GOCDB" section for more information about GOCDB and X509 certificates.
```

Comment:
```suggestion
```

I think this dates to a time before we integrated with EGI Check-In, it can be dropped.